cmake/FindOGRE.cmake

```diff
@@ -31,7 +31,7 @@
 #  RenderSystem_GL, RenderSystem_GL3Plus,
 #  RenderSystem_GLES, RenderSystem_GLES2,
 #  RenderSystem_Direct3D9, RenderSystem_Direct3D11
-#  Paging, Terrain, Volume, Overlay
+#  Paging, Terrain, Volume, Overlay, Bites
```

Comment:
Would be better to switch to the new upstream `/usr/lib64/OGRE/cmake/OGREConfig.cmake` eventually, but I'm trying to keep changes as minimal as possible for now (and with some luck to keep compat with 1.9 if possible).